commit cba94a1255aca4f98f8eb1c2d7f1a2f33266c49f (refs/changes/98/4498/2)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2018-08-01T14:07:31+00:00 (1 year, 2 months ago)
    
    joyent/node-verror#55 package.json repository does not point to Joyent
    Reviewed by: David Pacheco <dap@joyent.com>
    Approved by: David Pacheco <dap@joyent.com>

diff --git a/package.json b/package.json
index 79295c5..763a64f 100644
--- a/package.json
+++ b/package.json
@@ -5,7 +5,7 @@
 	"main": "./lib/verror.js",
 	"repository": {
 		"type": "git",
-		"url": "git://github.com/davepacheco/node-verror.git"
+		"url": "git://github.com/joyent/node-verror.git"
 	},
 	"dependencies": {
 		"assert-plus": "^1.0.0",
